src/lifecycle/events.js

```diff
@@ -28,7 +28,7 @@ function makeDelegateHandler (elem, handler, parsed) {
       if (matches(current, selector)) {
         readonly(e, 'currentTarget', current);
         readonly(e, 'delegateTarget', elem);
-        return handler(e);
+        return handler(elem, e);
```

Comment:
Those were documentation changes. I've tried to use `elem` in the internals consistently.